app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
```

Comment:
Code Style Issue: It's great that you are using parentheses for importing multiple classes. However, it's recommended to use one line if the line is not too long (less than 80 characters). This will make your code cleaner and easier to read. Here is an example: 

python
from app.errors import NotWearingMaskError, VaccineError
```